core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/component/ComponentConfigurationTest.java

```diff
@@ -208,6 +211,51 @@ void componentUpdate3Test() {
         assertEquals("dummy", fetchedComponentVersion3.get().getDescription());
     }
 
+    @Test
+    void componentUpdateParametersTest() {
+        ComponentConfiguration.getInstance().insert(component1);
+        ComponentConfiguration.getInstance().insert(component2);
+        ComponentConfiguration.getInstance().insert(component3);
+
+        ComponentParameter componentParameter1;
+        ComponentParameter componentParameter2;
+        ComponentParameter componentParameter3;
+        ComponentParameter componentParameter3Bis;
+        List<ComponentParameter> componentParameters1 = new ArrayList<ComponentParameter>();
+        List<ComponentParameter> componentParameters2 = new ArrayList<ComponentParameter>();
+        List<ComponentParameter> componentParameters3 = new ArrayList<ComponentParameter>();
```

Comment:
Not very important but you can write that like that too:
`List<ComponentParameter> componentParameters1 = new ArrayList<>();`

For initialization, the "sub-type" adapt itself to the declared one.